package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Wed Apr 10 07:55:26 UTC 2019 - David Diaz <dgonzalez@suse.com>
+
+- Fix partitioner to not have not doable operations after editing
+  a partition that initially contains an LvmPv not associated with
+  any LvmVg (bsc#1129663).
```

Comment:
NP: I find the explanation a little bit confusing. Maybe something like the following?

Partitioner: when editing a partition, remove its orphan LVM Physical Volume. This avoid no doable operations (bsc#1129663).

Reply:
>  I find the explanation a little bit confusing. 

I agree. However, I'll use the [Ancor's suggestion](https://github.com/yast/yast-storage-ng/pull/897#pullrequestreview-224884529), if you do not mind :wink: